src/Differ/MapDiffer.php

```diff
@@ -23,43 +25,17 @@
  */
 class MapDiffer implements Differ {
 
-	/**
-	 * @var bool
```

Comment:
I removed it cause I think it's just as easy to see this without the comment, having the comment just be clutter that makes other things harder to see. I agree these comments are often useful when the class is not this simple.